.envrc

```diff
@@ -0,0 +1,6 @@
+if ! has nix_direnv_version || ! nix_direnv_version 2.2.0; then
+  source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.2.0/direnvrc" "sha256-5EwyKnkJNQeXrRkYbwwRBcXbibosCJqyIUuz9Xq+LRc="
+fi
+GOPATH=`pwd`/.direnv/go
+PATH=${GOPATH}/bin:${PATH}
```

Comment:
wouldn't these get added by nix/flake?

Reply:
No, the flake can't refer to the original source location - only nix store locations after the flake is copied there.

Reply:
oh this is so we can completely isolate go workings to the repository?

Reply:
Yeah, mainly for things like automatically `go install`'d tools via the IDE